dashboard/app/models/user.rb

```diff
@@ -1089,6 +1089,8 @@ def next_unpassed_visible_progression_level(script)
   # Returns the next script_level for the next progression level in the given
   # script that hasn't yet been passed, starting its search at the last level we submitted
   def next_unpassed_progression_level(script)
+    # If a script is hidden, return the first level in the script where the user will see a "your teacher didn't expect you here" message.
+    return  script.stages.first.script_levels.first if script_hidden?(script)
```

Comment:
Ultimately this probably isn't what we want to do, so I've logged [LP-805](https://codedotorg.atlassian.net/browse/LP-805). @fontie715 and I can follow up with what we want the user experience to be if all of the stages are hidden for a script (should there be a note on the script overview? Do we not include that script on the homepage? Does the continue button disappear? etc)